docs/src/backend/15_testing.md

```diff
@@ -102,10 +102,10 @@ async fn health_check_works() {
         .to_request();
 
     let res = actix_web::test::call_service(&mut app, req).await;
-    
+
     assert!(res.status().is_success());
     assert_eq!(res.status(), StatusCode::OK);
-    let data = res.headers().get("version").and_then(|h| h.to_str().ok());
+    let data = res.headers().get("health-check").and_then(|h| h.to_str().ok());
```

Comment:
This is correct. As we're using the `version` header in our example. There's an error, though, in [line 42](https://github.com/BcnRust/devbcn-workshop/blob/f768bbf74f693603e166de9f90247b56fbb37fc5/docs/src/backend/15_testing.md#L42).

Reply:
My apologies.
I realized my mistake when I saw your comment.

Reply:
Oh, now I see that in ln. 42 the error was indeed intentional. Well closed. Thanks anyway for your time.